target/linux/rtl838x/files-5.4/arch/mips/rtl838x/prom.c

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
ERROR: do not use C99 // comments
WARNING: externs should be avoided in .c files